samples/tfm_integration/CMakeLists.inc

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019 Linaro
```

Comment:
This file should be named `<tfm-something>.cmake` instead, as it is a file being included as kind of boiler plating.

Reply:
renamed to `tfm_common.cmake`